finetune/input_pipeline.py

```diff
@@ -42,17 +42,25 @@ def _target_encoder(self):
         # Overridden by subclass to produce the right target encoding for a given target model.
         raise NotImplementedError
 
+    def _add_context_info_if_present(self, types, shapes):
+        if self.config.use_auxiliary_info:
+            TS = tf.TensorShape
+            types["context"] = tf.float32
```

Comment:
This will eventually have to be refactored if we intend to support types other than numeric inputs.